spec/lib/tasks/stagehand_tasks_spec.rake

```diff
@@ -0,0 +1,24 @@
+require 'rake'
+require 'rails_helper'
+
+Rails.application.load_tasks
+include Stagehand::Database
+
+describe "Stagehand Tasks" do
+  before do
+    @my_migration_version = '1'
+  end
+
+  describe "stagehand:migration" do
+    without_transactional_fixtures
+
+    it 'should migrate both tables' do
+      expect { Rake::Task['db:migrate'].invoke }.to change{ [production_database_versions.last, staging_database_version.last] }.to(["1", "1"])
```

Comment:
If these lines get too long, break them after `.invoke }` and indent two spaces.